user_templates/variables.cfg

```diff
@@ -182,6 +182,12 @@ variable_ercf_unload_on_cancel_print: False
 variable_ercf_unload_on_end_print: True
 variable_ercf_reset_stats_on_start_print: False
 
+################################################
+## Filter specific variables
+################################################
+## This section is only considered if a filter is available (and enabled)
+
+variable_filter_default_time_on_end_print = 600 # seconds
```

Comment:
The variable name is not correct and will lead to an error

Reply:
What exactly is the error? Too long?

Reply:
`variable_filter_default_time_on_end_print` is different than the one used in the macro : `variable_default_time_on_end_print`